storage/migrations/00_consensus.up.sql

```diff
@@ -25,14 +25,14 @@ CREATE DOMAIN public.eth_addr BYTEA CHECK(length(VALUE) = 20);
 CREATE TABLE chain.blocks
 (
   height     UINT63 PRIMARY KEY,
-  block_hash HEX64 NOT NULL,
-  time       TIMESTAMP WITH TIME ZONE NOT NULL,
-  num_txs    UINT31 NOT NULL,
+  block_hash HEX64 NOT NULL,  -- NULL in 32_block_meta.up.sql
```

Comment:
These comments are wrong  `32_block_meta.up.sql` does not make these fields `NULL`-able.